ruleguard/filters.go

```diff
@@ -104,6 +105,20 @@ func makeAddressableFilter(src, varname string) filterFunc {
 	}
 }
 
+func makeCustomVarFilter(src, varname string, env *quasigo.Env, fn *quasigo.Func) filterFunc {
+	return func(params *filterParams) matchFilterResult {
+		// TODO(quasilyte): what if bytecode function panics due to the programming error?
+		// We should probably catch the panic here, print trace and return "false"
+		// from the filter (or even propagate that panic to let it crash).
```

Comment:
Yep. But I'm not sure yet what should that recovered panic do.
Silently ignoring them is a bad thing.

I think the panic trace won't show which byte-compiled function failed. It could be a good idea to annotate the panic with this information and re-panic with extra info.